Xamarin.PropertyEditing.Mac/PropertyList.cs

```diff
@@ -119,6 +119,8 @@ public override bool ValidateProposedFirstResponder (NSResponder responder, NSEv
 				return true;
 			}
 
+			public override CGRect FrameOfOutlineCellAtRow (nint row) => CGRect.Empty;
```

Comment:
Why do we have a custom disclosure triangle?

Reply:
Because the current logic expects this item to handle the managed NextKeyResponder, so I wouldn't change this to avoid break things.